markdown.go

```diff
@@ -46,8 +46,6 @@ const (
 	AutoHeaderIDs                                 // Create the header ID from the text
 	BackslashLineBreak                            // Translate trailing backslashes into line breaks
 	DefinitionLists                               // Render definition lists
-	TOC                                           // Generate a table of contents
-	OmitContents                                  // Skip the main contents (for a standalone table of contents)
```

Comment:
Yeah, these extensions have nothing to do with parsing Markdown, so it makes sense to move them from here. 👍